Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1243,6 +1249,109 @@ async def check_and_handle_entitlement(text: str, user: dict, thread_id: str) ->
     return ''
 
 
+def handle_date_interval(fetch_interval: str, last_update_time):
```

Comment:
This looks like a generic utility method. Maybe it can be in CommonServerPython? Be made static?
Also - missing documentation.

Reply:
I'm open to moving it to common server, but is there a need? We have one that is similar, but looks back not forward. @yuvalbenshalom wdyt?